recipes/jpcre2/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class TomlPlusPlusConan(ConanFile):
+    name = "jpcre2"
+    homepage = "https://github.com/jpcre2/jpcre2"
+    description = "Header-only C++ wrapper for PCRE2 library."
+    topics = ("regex", "pcre2", "header-only", "single-header")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("pcre2/10.37")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
```

Comment:
```suggestion
        self.copy("LICENCE", dst="licenses", src=self._source_subfolder)
```

Reply:
For some reason licence file name is different in release archive and in the repository. In the repository it is "LICENCE" but in the archive it is "COPYING". As I can see other recipes preserve original file names. I think that I can rename this file if necessary but this change request should be double checked.

Reply:
Keep COPYING. I also saw few projects using LICENCE, it's the British form afaik

Reply:
COPYING is a symbolic link to LICENSE file in this repo.